containers/Containerfile

```diff
@@ -1,8 +1,8 @@
-FROM fedora:30
+FROM fedora:34
 
 MAINTAINER "Candlepin Team" <candlepin@redhat.com>
 
-RUN yum install --disablerepo=updates -y \
+RUN dnf install --disablerepo=updates -y \
```

Comment:
AFAIK we can continue to use yum as the cli packaging tooling indefinitely. Doesn't mean we can't accept this change but might not be necessary depends.